x-pack/test/saved_object_tagging/api_integration/security_and_spaces/apis/get_all.ts

```diff
@@ -35,7 +35,7 @@ export default function (ftrContext: FtrProviderContext) {
       authorized: {
         httpCode: 200,
         expectResponse: ({ body }) => {
-          expect(body).to.eql({
+          expect(body.tags.slice(0, 1)).to.eql({
```

Comment:
had to touch this test because with the new logic of adding tags for installed package assets, 2 new tags were added, because Synthetics package is being installed on startup

```
  actual: '{\n' +
    '  "tags": [\n' +
    '    {\n' +
    '      "color": "#FF00FF"\n' +
    '      "description": "Tag 1 in default space"\n' +
    '      "id": "default-space-tag-1"\n' +
    '      "name": "tag-1"\n' +
    '    }\n' +
    '    {\n' +
    '      "color": "#77CC11"\n' +
    '      "description": "Tag 2 in default space"\n' +
    '      "id": "default-space-tag-2"\n' +
    '      "name": "tag-2"\n' +
    '    }\n' +
    '    {\n' +
    '      "color": "#FFFFFF"\n' +
    '      "description": ""\n' +
    '      "id": "7833a370-0cfe-11ed-947f-438bf3d7f352"\n' +
    '      "name": "Elastic Synthetics"\n' +
    '    }\n' +
    '    {\n' +
    '      "color": "#FFFFFF"\n' +
    '      "description": ""\n' +
    '      "id": "77aa9d50-0cfe-11ed-947f-438bf3d7f352"\n' +
    '      "name": "Managed"\n' +
    '    }\n' +
    '  ]\n' +
    '}',
```